src/admin-dhis2/admin-dhis2.service.js

```diff
@@ -44,13 +44,18 @@
             removeServer: {
                 url: openlmisUrlFactory('/api/serverConfiguration/:id'),
                 method: 'DELETE'
+            },
+            getServerDatasets: {
+                url: openlmisUrlFactory('/api/serverConfiguration/:id/datasets'),
+                method: 'GET'
             }
         });
 
         this.getServerConfig = getServerConfig;
         this.addServer = addServer;
         this.editServer = editServer;
         this.removeServer = removeServer;
+        this.getServerDatasets = getServerDatasets;
```

Comment:
coukd you please add an empty line before this definition? I think we can 'group' these variables

Reply:
ok, sure

Reply:
```
/app/src/admin-dhis2/admin-dhis2.service.js
  58:1  error  Trailing spaces not allowed  no-trailing-spaces
```
There is such error after adding empty line FYI